function handleError(error) {
  console.log(error);
}

export { handleError };
